content/releases/q/release-2.13.0.md

```diff
@@ -0,0 +1,14 @@
+---
+url: "/releases/q-2.13.0/"
+date: 2022-09-26
+eol: 2022-03-26
```

Comment:
Is it like end of life in the past ?

Reply:
yes I checked what we did in 2.7.x, we did set for 6 months

Reply:
https://github.com/apache/camel-website/blob/main/content/releases/q/release-2.7.0.md

Reply:
date-fomats wise, it looks that in [2.7.x](https://github.com/apache/camel-website/blob/main/content/releases/q/release-2.7.0.md), eol > date.
So would we need to put `eol: 2023-03-26` ?

Reply:
oh okay. Nice catch